tests/libs/ajv.js

```diff
@@ -0,0 +1,7165 @@
+(function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.ajv = f()}})(function(){var define,module,exports;return (function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){
```

Comment:
I think in general we need to have copyright notices. Are these javascripts some sort of generated or pre-canned contents, or did you write them?

Reply:
This is the minimized version of the ajv library, as are the others. Typically the js libraries are minimized and obfuscated to cut down on transfer size, this is what i was asking about in AS the other day. No, I didn't write them, this is output from the postman-to-k6 tool and is included so that users wouldn't have to run the fixup described in the README. I'll find the license files and include them in a separate directory.